contributed_definitions/nyaml/NXxps.yaml

```
@@ -0,0 +1,520 @@
+        exists: optional
+        doc: |
+          In traditional surface science, the coordinate system is defined such that the positive z-axis
+          points along the normal of the sample stage, and the x- and y-axes lie in the plane of the sample stage.
+          However, in NeXus, a coordinate system that is the same as `McStas`_  is used. `xps_coordinate_system`
+          gives the user the opportunity to work in the traditional base coordinate system.
+          
+          .. _McStas: http://mcstas.org/
+          
+          .. image:: xps/xps_cs.png
+              :width: 40%
+        origin:
+          enumeration: [sample stage]
+        handedness:
+          enumeration: [left_handed]
+        z_direction:
+          enumeration: [sample stage normal]
+        x:
+          enumeration: [[-1, 0, 0]]
+        y:
+          enumeration: [[0, 1, 0]]
+        z:
+          enumeration: [[0, 0, 1]]
+        depends_on(NX_CHAR):
+          doc: |
+            Link to transformations defining the XPS base coordinate system,
+            which is defined such that the positive z-axis points along the sample stage normal, and the
+            x- and y-axes lie in the plane of the sample stage.
+        coordinate_system_transformations(NXtransformations):
+          doc: |
+            Set of transformations, describing the orientation of the XPS coordinate system
+            with respect to the beam coordinate system (.).
+    (NXinstrument):
+      doc:
+      - |
+        Description of the XPS spectrometer and its individual parts.
+      - |
+        xref:
+          spec: ISO 18115-1:2023
+          term: 12.58
+          url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.58
+      sourceTYPE(NXsource):
+        power(NX_FLOAT):
+          unit: NX_POWER
+          exists: recommended
+      beamTYPE(NXbeam):
+        depends_on:
+          exists: recommended
+          doc: |
+            Reference to the transformation describing the orientation of the beam
+            relative to a defined coordinate system.
+        transformations(NXtransformations):
+          exists: recommended
+          beam_azimuth_angle(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Azimuthal tilt of the beam from the y-direction towards the operator, defined
+              by the sample stage.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[0, 0, -1]]
+            \@depends_on:
+              enumeration: [beam_polar_angle_of_incidence]
+          beam_polar_angle_of_incidence(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Incidence angle of the beam with respect to the upward z-direction, defined by
+              the sample stage.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[1, 0, 0]]
```

From the graphic it looks like that you rotate along the x-axis. This would mean that you'd need to rotate around the  y-axis. If this is the case the vector is wrong, this is the correct one

```suggestion
              enumeration: [[0, 1, 0]]
```

---

agreed